zulipterminal/ui_tools/boxes.py

```diff
@@ -1412,13 +1418,21 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                 recipient_user_ids=[self.message['sender_id']],
             )
         elif is_command_key('MENTION_REPLY', key):
-            self.keypress(size, 'enter')
-            mention = f"@**{self.message['sender_full_name']}** "
-            self.model.controller.view.write_box.msg_write_box.set_edit_text(
-                mention)
-            self.model.controller.view.write_box.msg_write_box.set_edit_pos(
-                len(mention))
-            self.model.controller.view.middle_column.set_focus('footer')
+            stream_id = cast(int, self.stream_id)
```

Comment:
This doesn't seem right. The value you're casting to an int can be None, and the method you're passing it to happens to pass it to another method which treats None OK - but that's not guaranteed. Types are supposed to help us (via eg. mypy), but they can't if we do this.